app/views/challenges/show.html.erb

```diff
@@ -1,16 +1,23 @@
 <div class="container show-container">
-<%= link_to challenges_path do %>
+
+
+  <%= link_to challenges_path do %>
     <i class="fas fa-chevron-left"></i>
   <%  end %>
-
-<!-- <div class="col-12 mt-3 text-center">
-  <%#= image_tag("idea.png", size: "48x48") %>
-</div> -->
+<div class="col-12 text-center">
```

Comment:
uh what does this div do? seems to have nothing in it

Reply:
I'm honestly not sure... don't remember if I was trying something or removing something. Either way, not relevant anymore.